images/solr-drupal/9.Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG IMAGE_REPO
+FROM ${IMAGE_REPO:-lagoon}/commons as jumpstart
+
+ADD https://git.drupalcode.org/project/search_api_solr.git#4.x /search_api_solr
```

Comment:
Should this be pinned to a hash to prevent changes in the future? Or is this a departure from the old Solr ways of doing it, where we shipped old (but stable) solr conf?

Reply:
We shipped a stable config but never updated it. Ideally people would use their own config, with this as a fallback. There's a conversation in the linked issue.

My logic was that at this way the config stays current, but happy to consider other ways of doing it 😉

Reply:
i've come around on this a bit - the ADD directive allows a tag specification, so the maintenance is easier - so I tagged to the latest 4.3.2 release